recipes/tinyexr/all/conanfile.py

```diff
@@ -0,0 +1,68 @@
+import os
+import glob
+from conans import ConanFile, tools
+
+
+class TinyExrConan(ConanFile):
+    name = "tinyexr"
+    description = "Tiny OpenEXR image loader/saver library"
+    homepage = "https://github.com/syoyo/tinyexr"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+    topics = ("conan", "exr", "header-only")
+    options = {
+        "with_miniz": [True, False],
```

Comment:
```suggestion
        "with_zlib": [True, False],
```

Reply:
It uses miniz, not minizip.

Reply:
The option is called miniz, but you still require conan's zlib.

Reply:
zlib is only used when miniz is disabled.
I think we must respect the options imposed by the author of the library (https://github.com/syoyo/tinyexr#compile-flags). I personally think it is better to use `use_miniz` for `TINYEXR_USE_MINIZ`. With `use_zlib`, the consumer might wonder why there is no option for miniz after reading the documentation.

Reply:
Ok, I understand.
Then I see the following problem: you haven't added `miniz` as a requirement.
Also, maybe add the following option: `"with_z": ["zlib", "miniz"]` to make it clear to consumers that they have the choice between miniz and zlib.

Reply:
miniz is embedded. I can remove it but but it would be a huge patch (~6459 continuous lines)

Reply:
Is it a separate header file or embedded in the main header?
If it's embedded, can you surround the miniz code with `#if 0` and `#endif`?`

Reply:
It's embedded in the main header (See https://github.com/syoyo/tinyexr/blob/master/tinyexr.h#L591). Surrounding it with macros may work.

Reply:
The patch is a bit heavier than expected but it works.

Reply:
Great!
Now we will both sleep better 🤣